### PR TITLE
Add Google Analytics to installer

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8032/app_12644_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8032/HighFidelity-Beta-PR12644-2fe9f6df6f9bbfc6f5abb04818167c40fdd9aca5.dmg

File: cmake/templates/NSIS.template.in [hunk 1]
```diff
@@ -319,6 +319,87 @@ Function DownloadFile
...
+  !insertmacro CreateGUID $GAClientID
+
+  ; Push $0
+  ; ReadRegStr $GAClientID HKLM "@REGISTRY_HKLM_INSTALL_ROOT@\@CPACK_PACKAGE_INSTALL_REGISTRY_KEY@" "@CLIENT_ID_REG_KEY@"
+  ; ${REMatches} $0 "[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}" $GAClientID 0
+  ; ${If} $0 == false
+  ;   !insertmacro CreateGUID $GAClientID
+  ;   WriteRegStr HKLM "@REGISTRY_HKLM_INSTALL_ROOT@\@CPACK_PACKAGE_INSTALL_REGISTRY_KEY@" "@CLIENT_ID_REG_KEY@" $GAClientID
+  ; ${EndIf}
+  ; Pop $0
```
Comment (birarda) [inline cmake/templates/NSIS.template.in]: Keep this in the history so we can get it if we need, but pull it from this PR

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8045/app_12644_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8045/HighFidelity-Beta-PR12644-24ee97a643491208fc3dd7a3c88421c4fd87474b.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8061/app_12644_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8061/HighFidelity-Beta-PR12644-bbe98f61a6d35b09c629045c8a27947f74e6ed9d.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8146/app_12644_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8146/HighFidelity-Beta-PR12644-0117179fe4fffe3689456173e95babbbf9ec3844.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8146/HighFidelity-Beta-PR12644-0117179fe4fffe3689456173e95babbbf9ec3844.dmg

**Comment by Atlante45** [comment]
## Test Plan:
- [C5998](https://highfidelity.testrail.net/index.php?/cases/view/5998)
- [C5999](https://highfidelity.testrail.net/index.php?/cases/view/5999)

**Comment by birarda** [comment]
Let's make sure we have the GA ID for stable in place too for whenever RC66 goes out

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Clients installed successfully.

**Pull request merged**